modules/shared/configuration/ConfigPostCreation/config.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable */
 export enum configPostCreation {
   maxUploadedFiles = 4,
-  maxFileSizeInByte = 10000000, // 10MB
-  maxFileSizeInMegaByte = maxFileSizeInByte / 1000000,
+  maxFileSizeInByte = 1024, // 10MB
```

Comment:
1024 bytes is not equal to 1 megabyte, isn't it?

Reply:
@omarawad0  can you review the changes?